src/viur/core/config.py

```diff
@@ -593,7 +594,7 @@ class User(ConfigType):
     session_persistent_fields_on_login: Multiple[str] = ["language"]
     """If set, these Fields will survive the session.reset() called on user/login"""
 
-    session_persistent_fields_on_logout: Multiple[str] = ["language"]
+    session_persistent_fields_on_logout: Multiple[str] = []
```

Comment:
Did this change in accordance with @ArneGudermann to leave an empty (non-cookie-requiring) session after user logged out.